.github/workflows/CI.yml

```diff
@@ -85,16 +83,12 @@ jobs:
       - uses: julia-actions/setup-julia@v1
         with:
           version: '1'
-      - run: |
-          git config --global user.name name
-          git config --global user.email email
-          git config --global github.user username
       - run: |
           julia --project=docs -e '
-            using Pkg;
-            Pkg.develop(PackageSpec(path=pwd()));
-            Pkg.instantiate();
-            include("docs/make.jl");'
+            using Pkg
```

Comment:
I believe the semi-colons are needed here? Would this not just turn into:

> using Pkg Pkg.develop(PackageSpec(path=pwd())) Pkg.instan.....

Reply:
@omus made a comment on another PR a long time ago about how we don't need the semi colons. So I've been removing them since they aren't necessary.